src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super.setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: circle, area: " + getArea()
```

Comment:
<img width="821" alt="Screen Shot 2023-01-03 at 8 30 59 PM" src="https://user-images.githubusercontent.com/90692308/210419292-f40dc3c8-d9a3-4d64-b2e3-f8d1f982b3f1.png">
Be careful

Reply:
Could you please give me a hint? I didn't understand it from the beginning. If I create a Drawable interface, the method draw() in it can't have a body so I have to override it anyway. Am I wrong?

Reply:
В задании сказано, что метод draw должен напечатать информацию про фигуру используя System.out.println(). Здесь не нужно ничего возвращать